tests/integration/utils.py

```diff
@@ -60,10 +60,12 @@
     "https://raw.githubusercontent.com/CycloneDX/specification/1.4/schema/bom-1.4.schema.json"
 )
 
+DEFAULT_OUTPUT_DIR = Path("cachi2-output")
+DEFAULT_ENV_FILE = Path("cachi2.env")
```

Comment:
We're only ever going to use this in a single test primitive related to the `generate-env` command, so we can skip the constant and hardcode the path in the invocation, it's fine.